celery_tester.py

```diff
@@ -0,0 +1,38 @@
+import pytest
```

Comment:
helper function for launching celery for testings. only works some of the time due to this issue below. if u do the two steps manually rather than launching as a subprocess it works fine, but requires manual setup

Reply:
it works fine 99% of the time on CI/CD: because Circle uses docker compose, the CI/CD tests run on a far more sophisticated worker setup that includes dedicated workers for txn processing